lib/models/services-schema.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+var Genesis = require("genesis");
+var Joi     = require("joi");
+
+module.exports = Genesis.create("services",	{
```

Comment:
The first parameter to `Genesis.create` is the name of the model. In this case, our model's name should be `instance`.